susemanager/bin/mgr-setup

```diff
@@ -2,8 +2,8 @@
 
 DISTRIBUTION_ID="$(source /etc/os-release && echo ${ID})"
 case ${DISTRIBUTION_ID} in
-    sles)      JVM='/usr/lib64/jvm/jre-1.8.0-ibm/bin/java';;
-    opensuse)  JVM='/usr/lib64/jvm/jre-1.8.0-openjdk/bin/java';;
+    sles)      JVM='/usr/lib64/jvm/jre-11-openjdk/bin/java';;
+    opensuse)  JVM='/usr/lib64/jvm/jre-11-openjdk/bin/java';;
```

Comment:
When we are using the same version for sles and opensuse, I wonder if we need the case at all. On the other hand, in the future we might want to use different versions for sles and opensuse again, so I would leave it as it is now. Looks good to me.